packages/wallet-sdk/src/relay/WalletLinkRelay.ts

```diff
@@ -133,17 +121,19 @@ export class WalletLinkRelay extends WalletSDKRelayAbstract {
   public subscribe() {
     const session = Session.load(this.storage) || new Session(this.storage).save();
 
-    const connection = new WalletLinkConnection(
+    const { linkAPIUrl, diagnostic } = this;
+    const connection = new WalletLinkConnection({
       session,
-      this.linkAPIUrl,
-      this.listener,
-      this.diagnostic
-    );
+      linkAPIUrl,
+      diagnostic,
+      listener: this,
+    });
 
+    const { version, darkMode } = this.options;
     const ui = this.options.uiConstructor({
-      linkAPIUrl: this.options.linkAPIUrl,
-      version: this.options.version,
-      darkMode: this.options.darkMode,
+      linkAPIUrl,
+      version,
+      darkMode,
```

Comment:
@bangtoven if we're not using `version` and `darkMode` anywhere else why don't we just:
```
version: this.options.version
```
?

Reply:
@willn-cb i think i did that for the consistency and code readability. open to suggestions.

Reply:
![image](https://github.com/coinbase/coinbase-wallet-sdk/assets/966662/a8a6bf8a-5f51-423a-921e-dcf051fa582e)

Reply:
without git diffing, it actually looks like this ^ @willn-cb

Reply:
Yeah fair that's probably more readable